README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
The second and third lines in your description make sense to me, however you keep referring to a __Simulated Robot__ component as if it was another process that runs outside the client or the server, perhaps I'm misunderstanding this.  For clarification purposes, what I think the presumed  architecture we both envision is as follows:
- Server Machine:
  - Runs perceptions nodes such as surface detection
  - Runs motion planning nodes such as the process path planner and free motion planner (moveit).
- Client Machine:
 - Runs Rviz for user front-end and visualization only.
 - Runs either **industrial_robot_simulator** or **robot_driver** node depending on whether it is on _sim_ or _real_ mode.
 - Runs either **generate_point_cloud** node or **Kinect sensor driver** node also based on the mode being used.

I might have missed some other important but not critical nodes, but this is how I understand the architecture of our distributed blending application will be.  This not by any means set in stone but I just want to make sure that we are on the same page about this.  Let me know if this is not what UTD has in mind.

Furthermore, I just wanted to stress the point that Rviz doesn't control any part of the process, it's there merely for visualization and user interaction.  The simulated robot you see there is just a fancy 3d animation, there is no robot code running inside of rviz that may affect the rest of the process.